storage/client_test.go

```diff
@@ -51,6 +51,15 @@ import (
 	"github.com/gogo/protobuf/proto"
 )
 
+func rg1(s *storage.Store) client.Sender {
+	return client.Wrap(s, func(ba roachpb.BatchRequest) roachpb.BatchRequest {
+		if ba.RangeID == 0 {
+			ba.RangeID = 1
```

Comment:
I did that in the internal tests, but this is `storage_test`. I had intermittently introduced `(*Store).TestSender()` but didn't want to leak obvious test code into the public interface. Agreed that this looks clumsier than it should be - how would you go about it?